pkgs/applications/networking/mailreaders/aerc/default.nix

```diff
@@ -66,6 +61,7 @@ buildGoModule rec {
       "$out/share/aerc/filters:${lib.makeBinPath [ ncurses ]}"
     wrapProgram $out/share/aerc/filters/html --prefix PATH ":" \
       ${lib.makeBinPath [ w3m dante ]}
+    wrapPythonProgramsIn $out/share/aerc/filters "$out $pythonPath"
```

Comment:
Is this actually necessary? I didn't saw any Python programs in this PATH.

Reply:
I guess you can remove it.

Reply:
Done.